packages/peregrine/lib/store/reducers/cart.js

```diff
@@ -2,12 +2,16 @@ import { handleActions } from 'redux-actions';
 
 import actions from '../actions/cart';
 import checkoutActions from '../actions/checkout';
+import { Util } from '../../index';
+
+const { BrowserPersistence } = Util;
+const storage = new BrowserPersistence();
 
 export const name = 'cart';
 
 export const initialState = {
     addItemError: null,
-    cartId: null,
+    cartId: storage.getItem('cartId'),
```

Comment:
This is a pattern also used by the `user` reducer to ensure we synchronize the state on load with local storage.